### DISCUSSION TITLE
[Help]: Does using multiple sliders reduce a website's performance?

### DISCUSSION THREAD

**Discussion by notrealdev** [davidjerleke/embla-carousel#979] [Help] [answered]
### Summary

Hi, I am developing a website similar to [calvin](https://www.calvinklein.us/en/fall-collection/womens-fall-clothing). I plan to use Embla, which might be feasible, but I’m concerned about potential performance issues.

What do you guys think about this if I have thousands of products? Each product item will have a slider.

### If applicable, which variants of Embla Carousel are relevant to this conversation?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by meirroth** [comment, marked as answer]
@notrealdev We've got many sliders on our homepage and performance is fine. [Lazy loading](https://github.com/davidjerleke/embla-carousel/discussions/835#discussioncomment-9188550) slides helps!

**Reply by notrealdev** [reply]
Hi @meirroth can you share your website? I'm just curious to see it live.

**Reply by meirroth** [reply]
Sure: https://torahlive.com/ Each category row is another slider.

**Reply by notrealdev** [reply]
The website is very nice. I see that, thank you!

**Reply by meirroth** [reply]
@notrealdev Thanks! 😃 
Essentially, try to reduce number of DOM elements on the page. Only rendering what's visible.

**Reply by davidjerleke** [reply]
@notrealdev Just to add to the conversation — you can also lazy load entire carousels and unmount them when they leave the viewport.
